src/frontend/src/pages/TeamsPage/TeamMembersPageBlock.tsx

```diff
@@ -179,32 +179,32 @@ const TeamMembersPageBlock: React.FC<TeamMembersPageBlockProps> = ({ team }) =>
 
   const NonEditingHeadView = () => (
     <Grid container>
-      <Grid item>
+      <Grid item xs={11}>
         <DetailDisplay label="Head" content={fullNamePipe(team.head)} />
       </Grid>
-      <Grid item mt={-1}>
-        {hasPerms && <IconButton children={<Edit />} onClick={() => setIsEditingHead(true)} />}
+      <Grid item xs={1} mt={-1} display={'flex'} justifyContent={'flex-end'}>
+        {hasPerms && <IconButton children={<Edit />} onClick={() => setIsEditingLeads(true)} />}
```

Comment:
Should this be `setIsEditingHead` instead of leads?

Reply:
Yeah, I forgot to change that